README.md

```diff
@@ -87,6 +87,23 @@ post_install do |installer|
 end
 ```
 
+When you use M1 Macs, you may get pod install failling when adding or upgrading `livekit_client`, you can use the steps below to fix it.
+
+1. Set the ios version on `Podfile` to `12.1`
+
+```xml
+platform :ios, '12.1'
+```
+
+2. Delete `Podfile.lock`
+
+3. Run pod install again
+
+```bash
+cd ios
+arch -x86_64 pod install --repo-update
+```
+
```

Comment:
```suggestion
```

Reply:
**Running pod install --repo-update in m1 mac will report an error**

<img width="659" alt="image" src="https://user-images.githubusercontent.com/17733746/195532407-82c545a6-a00c-429c-a472-ad19e1e4a195.png">

**need to add prefix `arch -x86_64`**

<img width="696" alt="image" src="https://user-images.githubusercontent.com/17733746/195532879-b5e01d79-8a79-4a0e-a1f4-ecd6c05c2357.png">

Reply:
The latest `cocoapods` no longer require the command prefix `arch -x86_64` on m1, so your suggestion is correct.
https://stackoverflow.com/questions/64901180/how-to-run-cocoapods-on-apple-silicon-m1